lib/manager/composer/artifacts.spec.ts

```diff
@@ -367,4 +368,89 @@ describe('manager/composer/artifacts', () => {
     ).not.toBeNull();
     expect(execSnapshots).toMatchSnapshot();
   });
+
+  it('installs before running the update when symfony flex is installed', async () => {
+    fs.readLocalFile.mockResolvedValueOnce(
+      '{"packages":[{"name":"symfony/flex","version":"1.17.1"}]}'
+    );
+    const execSnapshots = mockExecAll(exec);
+    fs.readLocalFile.mockResolvedValueOnce('{ }');
+    git.getRepoStatus.mockResolvedValue({
```

Comment:
Should also use once?

Reply:
That works as well, I took the liberty to change all occurrences in the `artifacts.spec` file.